content/zh/docs/concepts/security/index.md

```diff
@@ -1,850 +1,852 @@
-开始在已部署的服务中使用 Istio 安全功能。请访问我们的[安全任务](/zh/docs/tasks/security/)，
-以获取有关使用安全功能的详细说明。
-
-## 高层架构 {#high-level-architecture}
-
-Istio 中的安全性涉及多个组件：
-
-- 用于密钥和证书管理的证书颁发机构（CA）
-- 配置 API 服务器分发给代理：
-
-    - [认证策略](/zh/docs/concepts/security/#authentication-policies)
-    - [授权策略](/zh/docs/concepts/security/#authorization-policies)
-    - [安全命名信息](/zh/docs/concepts/security/#secure-naming)
-
-- Sidecar 和边缘代理作为[策略执行点](https://www.jerichosystems.com/technology/glossaryterms/policy_enforcement_point.html)（PEP）
-  以保护客户端和服务器之间的通信安全。
-- 一组 Envoy 代理扩展，用于管理遥测和审计
-
-控制面处理来自 API server 的配置，并且在数据面中配置 PEP。
-PEP 用 Envoy 实现。下图显示了架构。
-
-{{< image width="75%"
-    link="./arch-sec.svg"
-    caption="安全架构"
-    >}}
-
-在下面的部分中，我们将详细介绍 Istio 安全功能。
-
-## Istio 身份 {#istio-identity}
-
-身份是任何安全基础架构的基本概念。在工作负载间通信开始时，
-双方必须交换包含身份信息的凭证以进行双向验证。在客户端，
-根据[安全命名](/zh/docs/concepts/security/#secure-naming)信息检查服务器的标识，
-以查看它是否是工作负载授权的运行程序。在服务器端，
-服务器可以根据[授权策略](/zh/docs/concepts/security/#authorization-policies)确定客户端可以访问哪些信息，
-审计谁在什么时间访问了什么，根据他们使用的工作负载向客户收费，
-并拒绝任何未能支付账单的客户访问工作负载。
+- 
+  
+- ---
+  title: 安全
+  description: 描述 Istio 的授权与认证功能。
+  weight: 30
+  keywords: [security,policy,policies,authentication,authorization,rbac,access-control]
+  aliases:
+      - /zh/docs/concepts/network-and-auth/auth.html
+      - /zh/docs/concepts/security/authn-policy/
+      - /zh/docs/concepts/security/mutual-tls/
+      - /zh/docs/concepts/security/rbac/
+      - /zh/docs/concepts/security/mutual-tls.html
+      - /zh/docs/concepts/policies/
+  owner: istio/wg-security-maintainers
+  test: n/a
+  ---
+
+  将单一应用程序分解为微服务可提供各种好处，包括更好的灵活性、
+  可伸缩性以及服务复用的能力。但是，微服务也有特殊的安全需求：
+
+  - 为了抵御中间人攻击，需要流量加密。
+  - 为了提供灵活的服务访问控制，需要双向 TLS 和细粒度的访问策略。
+  - 要确定谁在什么时候做了什么，需要审计工具。
+
+  Istio Security 尝试提供全面的安全解决方案来解决所有这些问题。
+  本页概述了无论您在何处运行服务都能使用 Istio 的安全功能来保护这些服务。
+  特别是 Istio 安全性可以减轻针对您的数据、端点、通信和平台的内外威胁。
+
+  {{< image width="75%"
+      link="./overview.svg"
+      caption="安全概述"
+      >}}
+
+  Istio 安全功能提供了强大的身份、强大的策略、透明的 TLS 加密、
+  认证/授权/审计（AAA）工具来保护您的服务和数据。Istio 安全的目标是：
+
+  - 默认安全：应用程序代码和基础设施无需更改
+  - 深度防御：与现有安全系统集成以提供多层防御
+  - 零信任网络：在不受信任的网络上构建安全解决方案
+
+  请访问我们的[双向 TLS 迁移](/zh/docs/tasks/security/authentication/mtls-migration/)相关文章，
+  开始在已部署的服务中使用 Istio 安全功能。请访问我们的[安全任务](/zh/docs/tasks/security/)，
+  以获取有关使用安全功能的详细说明。
+
+  ## 高层架构 {#high-level-architecture}
+
+  Istio 中的安全性涉及多个组件：
+
+  - 用于密钥和证书管理的证书颁发机构（CA）
+  - 配置 API 服务器分发给代理：
+
+      - [认证策略](/zh/docs/concepts/security/#authentication-policies)
+      - [授权策略](/zh/docs/concepts/security/#authorization-policies)
+      - [安全命名信息](/zh/docs/concepts/security/#secure-naming)
+
+  - Sidecar 和边缘代理作为[策略执行点](https://www.jerichosystems.com/technology/glossaryterms/policy_enforcement_point.html)（PEP）
+    以保护客户端和服务器之间的通信安全。
+  - 一组 Envoy 代理扩展，用于管理遥测和审计
+
+  控制面处理来自 API server 的配置，并且在数据面中配置 PEP。
+  PEP 用 Envoy 实现。下图显示了架构。
+
+  {{< image width="75%"
+      link="./arch-sec.svg"
+      caption="安全架构"
+      >}}
+
+  在下面的部分中，我们将详细介绍 Istio 安全功能。
```

Comment:
Totally wrong! Why adding 2 spaces to each line?

Reply:
sorry,md tool automatic formatting